.github/workflows/build-and-deploy-prod.yml

```diff
@@ -57,14 +57,42 @@ jobs:
                 branch_name: master
                 output_dir: dist
                 cname_url: p2p.deriv.com
-    send_slack_success:
-      name: Send Release Slack notification success
-      runs-on: ubuntu-latest
-      if: success()
-      needs: [build_to_cloudflare_pages]
-      steps:
-        - name: Send Slack Notification
-          uses: "deriv-com/shared-actions/.github/actions/send_slack_notification@master"
-          with:
-            SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-            MESSAGE: "Production Release succeeded for p2p.deriv.com with version ${{ github.head_ref }}"
+    publish_to_vercel:
+        runs-on: ubuntu-latest
+        environment: Production
+        steps:
+            - name: Upload to vercel
+              uses: 'deriv-com/shared-actions/.github/actions/vercel_DR_publish@master'
+              with:
+                VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+                VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+                VERCEL_TOKEN: ${{ secrets.VERCEL_API_TOKEN }}
+                ENVIRONMENT: Production
+                VERCEL_SCOPE: deriv
+                ALIAS_DOMAIN_URL: 'p2p-dr.deriv.com'
+            - name: Send Slack Notification on Vercel Publish failure
+              uses: "deriv-com/shared-actions/.github/actions/send_slack_notification@master"
```

Comment:
```suggestion
              uses: "deriv-com/shared-actions/.github/actions/send_slack_notification@master"
              if: steps.dr_action.conclusion == 'failure'
```

Reply:
This is to make sure that only if the the previous job fails we are sending the slack message.